packages/amplify_api/android/build.gradle

```diff
@@ -0,0 +1,45 @@
+group 'com.amazonaws.amplify.amplify_api'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.3.50'
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 28
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+    defaultConfig {
+        minSdkVersion 21
+    }
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.amplifyframework:aws-api:1.6.0"
+    implementation "com.amplifyframework:aws-api-appsync:1.6.0"
```

Comment:
> Is there anything we need to do to support pagination for list type query?

Not for these APIs where the GraphQL document and variables need to be provided, since the customer explicitly decides what they want in the selection set, such as the pagination token.
In the future when we support model based APIs, we will need to handle it.